docs/development/classlib/classlib-manual.rst

```diff
@@ -0,0 +1,72 @@
+.NET Core Class Library Manual
+==============================
+
+.NET Core is a modern implementation of .NET. It includes many familiar .NET APIs, which are supported on Linux, OS X, Windows and other operating systems. It is both an implementation but also a standard API definition that can (and is) supported by other versions of .NET. 
```

Comment:
"that can [be] (and is)"?

I'm also a little confused about how something can be both an implementation and a standard API definition.  If I understand this correctly, the API definition is sort of like when you `f-12` in Visual Sudio and you see the API calls available to you?  It may be worthwhile to explain that it's an implementation of an API which, when included as a package, can also be viewed as an API.  To me, "API definition" means an interface and not necessarily functionality.